components/talksPage/talksPage.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import style from './talksPage.module.scss';
+import data from '../../assets/videos.json';
+import YouTube from 'react-youtube';
+
+function TalksPage() {
+  const talksList = data;
+  const [currSrc, setSrc] = useState('GHbNaDSWUX8');
+
+  const calcArrowSide = (curr, side, length) => {
```

Comment:
can you explain a bit about this function?

Reply:
It calculates the arrows on the side of the carousel.